docs/variants/asus_kgpe_d16/dasharo-compatibility/generic-test-setup.md

```diff
@@ -0,0 +1,19 @@
+# Dasharo Compatibility: generic test setup
+
+## Test setup
+
+Test setup is a set of procedures to be executed before the test execution.
+Typically, the same setup can be reused by multiple test cases, so there is no
+need to execute the setup actions before each independent case.
+
+### Generic test setup
+
+#### Firmware
+
+1. Obtain `FIRMWARE` binary
+    1. you can download it from [releases](../releases.md)
+       section
+    1. or you can build one yourself as shown in the
```

Comment:
Lepiej chyba, żeby zdanie nie zaczynało się od or, podpunkty też lepiej wyglądają napisane z dużej litery. Problem też jest z numeracją. Może obie opcje lepiej będzie ująć w jednym zdaniu i punkcie.

Reply:
Changed to subsections